app/controllers/ShutMucrController.scala

```diff
@@ -48,10 +53,18 @@ class ShutMucrController @Inject()(
       .fold(
         formWithErrors => Future.successful(BadRequest(shutMucrPage(formWithErrors))),
         shutMucr =>
-          submissionService.submitShutMucrRequest(formData = shutMucr).map { _ =>
-            Redirect(controllers.routes.ShutMucrConfirmationController.displayPage())
-              .flashing(Flash(Map(FlashKeys.MUCR -> shutMucr.mucr)))
+          submissionService.submitShutMucrRequest(shutMucr).map {
+            case ACCEPTED =>
+              Redirect(controllers.routes.ShutMucrConfirmationController.displayPage())
+                .flashing(Flash(Map(FlashKeys.MUCR -> shutMucr.mucr)))
+            case _ => handleError("Unable to submit Shut a Mucr Consolidation request")
         }
       )
   }
+
+  private def handleError(logMessage: String)(implicit request: Request[_]): Result = {
```

Comment:
I think we can move this method to `ErrorHandler` it's exactly the same in few controllers

Reply:
It depends. If we mvoe it to `ErrorHandler` then we would need to pass logger as an argument. Having log errors all being called by one object is hard to track.
I know that it breaks DRY rules, but the only purpose of this method is actually to log the error. I'll leave it for now.